sage/pallet/src/lib.rs

```diff
@@ -166,20 +168,21 @@ pub mod pallet {
 		StorageMap<_, Identity, AssetIdOf<T, I>, (AccountIdOf<T>, AssetOf<T, I>)>;
 
 	#[pallet::storage]
-	pub type AssetOwners<T: Config<I>, I: 'static = ()> = StorageDoubleMap<
-		_,
-		Identity,
-		AccountIdOf<T>,
-		Identity,
-		SeasonIdOf<T, I>,
-		BoundedAssetIdsOf<T, I>,
-		ValueQuery,
-	>;
+	pub type AssetOwners<T: Config<I>, I: 'static = ()> =
+		StorageDoubleMap<_, Identity, AccountIdOf<T>, Identity, AssetIdOf<T, I>, (), ValueQuery>;
+
+	#[pallet::storage]
+	pub type AmountAssetsOwned<T: Config<I>, I: 'static = ()> =
+		StorageMap<_, Identity, AccountIdOf<T>, u8, ValueQuery>;
```

Comment:
I am not sure if you are ready for the ultimate nitpick:

```suggestion
	pub type AssetsOwnedCount<T: Config<I>, I: 'static = ()> =
		StorageMap<_, Identity, AccountIdOf<T>, u8, ValueQuery>;
```

In English, amount is used for non-discrete things, but if it is discrete, you use count. 🙈

Reply:
Fixed!